Assets/MRTK/Providers/WindowsMixedReality/Shared/Editor/ConfigurationChecker/Microsoft.MixedReality.Toolkit.Providers.WMR.ConfigCheck.asmdef

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.ConfigCheck",
+    "name": "Microsoft.MixedReality.Toolkit.Providers.WMR.ConfigCheck",
```

Comment:
Do we expect other people to take dependencies on this config check assembly? If so this would be a breaking change we'd want to add to the releasenotes.md.

Why was this changed in this PR?

Reply:
We do not expect any customers to depend upon this assembly. It is used internally by MRTK.

It was changed here due to the path length getting too long and breaking compile of the Windows Mixed Reality provider

Reply:
In fact, there were some unnecessary assemblies that have been folded in this PR. :smile: